tests/components/withings/test_binary_sensor.py

```diff
@@ -17,12 +17,11 @@ async def test_binary_sensor(
     hass: HomeAssistant,
     withings: AsyncMock,
     disable_webhook_delay,
-    config_entry: MockConfigEntry,
+    webhook_config_entry: MockConfigEntry,
```

Comment:
For all unit test updates I only see the webhook config referenced. Are the differences with the polling implementation covered?